### PR TITLE
doc: update 'enable_ivshmem'

### PR DESCRIPTION
**Pull request by yonghuah**
Add notification (doorbell) usage for ivshmem devices.

Signed-off-by: Yonghua Huang <yonghua.huang@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/enable_ivshmem.rst [hunk 1]
```diff
@@ -66,6 +66,29 @@ enable it using the  :ref:`acrn_configuration_tool` with these steps:
 
 - Build the XML configuration, refer to :ref:`getting-started-building`
 
+ivshmem notification usage
```
Comment (gvancuts) [inline doc/tutorials/enable_ivshmem.rst:69]: Wouldn't
```suggestion
ivshmem notification mechanism
```
be a bit better?

**Review by gvancuts — APPROVED** [review]
LGTM from a content perspective.

File: doc/tutorials/enable_ivshmem.rst (hunk 1 above)
Comment (yonghuah) [reply doc/tutorials/enable_ivshmem.rst]: @gvancuts , updated , thank you.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Review by dbkinder — APPROVED** [review]
I fixed some formatting issues directly, LGTM.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**